ctf/tests/core_suite.lua

```diff
@@ -0,0 +1,91 @@
+local lunatest = require("lunatest")
+--minetest environment
+vector = {}
+minetest = {}
+function minetest.after(val, fx)
+end
```

Comment:
Would be good to share this (minetest dummy) across suites. But this could be a future update